main.js

```diff
@@ -81,23 +104,49 @@ function updateLatestGuess() {
   resetButton.disabled = true;
 }
 
+function updateRange() {
+  minSpan.innerText = minRange.value;
+  maxSpan.innerText = maxRange.value;
+  var min = parseInt(minRange.value);
+  var max = parseInt(maxRange.value);
+  correctNumber = getRandomRange(min, max);
+  minRange.value = "";
+  maxRange.value = "";
+  updateButton.disabled = true;
+}
+
 function checkGuess(guessInput, feedbackMessage) {
   var currentGuess = parseInt(guessInput.value);
+  var tooHigh = "that's too high";
+  var tooLow = "that's too low";
+  var perfectGuess = "BOOM!";
```

Comment:
Is there a reason you made these variables?  This just seems cluttered to me

Reply:
I was thinking if a person guess the right answer, the perfectGuess variable will be used to add the Winner's Output card on the right side on another function.  If there is another of doing this, then you can change it back to the original. I don't mind.  I wasn't quite sure of how to start with that part, so I did that is my first idea.

Reply:
These variables are stored locally within the function, so they can only be used by the function.  Let's just worry about one feature at a time.

Reply:
Thanks for letting me know that.  That will be helpful for me in the future.  I'm still working on my javascript skills. =)